--- a/CMakeLists.txt	2020-11-11 03:01:53.000000000 +0100
+++ b/CMakeLists.txt	2021-02-07 12:45:31.000000000 +0100
@@ -22,14 +22,14 @@
 SET(FAIRROOT_MINOR_VERSION 4)
 SET(FAIRROOT_PATCH_VERSION 2)
 
-if(APPLE)
-  if(NOT DEFINED CMAKE_C_COMPILER)
-    set(CMAKE_C_COMPILER clang)
-  endif()
-  if(NOT DEFINED CMAKE_CXX_COMPILER)
-    set(CMAKE_CXX_COMPILER clang++)
-  endif()
-endif()
+# if(APPLE)
+#   if(NOT DEFINED CMAKE_C_COMPILER)
+#     set(CMAKE_C_COMPILER clang)
+#   endif()
+#   if(NOT DEFINED CMAKE_CXX_COMPILER)
+#     set(CMAKE_CXX_COMPILER clang++)
+#   endif()
+# endif()

# Set name of our project to "FAIRROOT"
# Has to be done after check of cmake version
