--- a/Framework/TestWorkflows/src/o2DummyWorkflow.cxx	2021-02-07 16:48:22.000000000 +0100
+++ b/Framework/TestWorkflows/src/o2DummyWorkflow.cxx	2021-02-07 16:51:21.000000000 +0100
@@ -78,7 +78,7 @@
     {OutputSpec{{"summary"}, "TPC", "SUMMARY"}},
     AlgorithmSpec{[](ProcessingContext& ctx) {
       auto& tpcSummary = ctx.outputs().make<Summary>(OutputRef{"summary"}, 1);
-      tpcSummary.at(0).inputCount = ctx.inputs().size();
+      tpcSummary[0].inputCount = ctx.inputs().size();
     }},
     {ConfigParamSpec{"some-cut", VariantType::Float, 1.0f, {"some cut"}}},
   };
@@ -91,7 +91,7 @@
     },
     AlgorithmSpec{[](ProcessingContext& ctx) {
       auto& itsSummary = ctx.outputs().make<Summary>(OutputRef{"summary"}, 1);
-      itsSummary.at(0).inputCount = ctx.inputs().size();
+      itsSummary[0].inputCount = ctx.inputs().size();
     }},
     {ConfigParamSpec{"some-cut", VariantType::Float, 1.0f, {"some cut"}}},
   };
