--- a/Framework/Core/src/runDataProcessing.cxx	2021-01-21 09:51:28.000000000 +0100
+++ b/Framework/Core/src/runDataProcessing.cxx	2021-02-07 16:15:09.000000000 +0100
@@ -106,9 +106,10 @@
 #elif __has_include(<linux/getcpu.h>)
 #include <linux/getcpu.h>
 #elif __has_include(<cpuid.h>)
-#include <cpuid.h>
-#define CPUID(INFO, LEAF, SUBLEAF) __cpuid_count(LEAF, SUBLEAF, INFO[0], INFO[1], INFO[2], INFO[3])
-#define GETCPU(CPU)                                 \
+#if !defined(__arm64__)
+  #include <cpuid.h>
+  #define CPUID(INFO, LEAF, SUBLEAF) __cpuid_count(LEAF, SUBLEAF, INFO[0], INFO[1], INFO[2], INFO[3])
+  #define GETCPU(CPU)                                 \
   {                                                 \
     uint32_t CPUInfo[4];                            \
     CPUID(CPUInfo, 1, 0);                           \
@@ -121,6 +122,12 @@
     if (CPU < 0)                                    \
       CPU = 0;                                      \
   }
+#else
+  #define GETCPU(CPU)                                 \
+    {                                                 \
+        CPU = 0;                                      \
+    }
+#endif
 #endif
 
 using namespace o2::monitoring;
